e2e/windows/e2e-create-windows-nodepool.sh

```diff
@@ -6,9 +6,8 @@ source e2e-helper.sh
 
 log "Starting to create windows nodepool"
 
-if [[ "$RESOURCE_GROUP_NAME" == *"windows"*  ]]; then
-    RESOURCE_GROUP_NAME="$RESOURCE_GROUP_NAME"-"$WINDOWS_E2E_IMAGE"-v2
-fi
+K8S_VERSION=$(echo $KUBERNETES_VERSION | tr '.' '-')
+RESOURCE_GROUP_NAME="$RESOURCE_GROUP_NAME"-"$WINDOWS_E2E_IMAGE"-"$K8S_VERSION"
```

Comment:
will this impact the existing pipelines/tests? I think that @junjiezhang1997 can explain why the old code uses v2.

Reply:
This won't impact existing pipelines/tests. v2 is used to avoid impacting previous pipelines and current "$K8S_VERSION" can help to avoid existing pipelines.